packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -67,6 +97,13 @@ export type GetSelector<Component extends (props: any) => JSX.Element | undefine
   ? typeof errorBarSelector
   : undefined;
 
+/**
+ * Get the selector for a specified component.
+ *
+ * Used by the top-level packlet to implement merged {@link usePropsFor}.
+ *
```

Comment:
This one should be exposed as getCallingSelector, which could be used like this: [Example](https://github.com/PorterNan/ui-toolkit-quickstart-composites/blob/eacf1996b997257fc5c97768babc44793001f016/src/composites/CallComposite/ConfigurationScreen.tsx#:~:text=const%20options%20%3D%20useSelector(getCallingSelector(OptionsButton))%3B)

Same as getSelector for chat-binding

Reply:
I want to discuss this one with you IRL, but this PR is a bad place for it, so adding it back.